src/styles/lyrics.scss

```diff
@@ -13,9 +15,12 @@
     display: inline-block;
     width: fit-content;
     margin: 0.1em;
-    font-size: 30px;
+    font-size: 1.5em;
```

Comment:
This seems small on desktop and `rem` is probably a better unit for font sizes.

Reply:
Would you require me to increase the font size in that case?

Reply:
Yes, the font seems pretty small on desktop compared to what it was previously.